.env-example

```diff
@@ -61,6 +61,8 @@ export PAID_FALLBACK_PERCENT=90
 export PG_DATABASE_NAME='code_fund_ads_development'
 export PG_TIMEOUT_IN_SECONDS=3
 export PG_URL=
+export PG_USERNAME=
+export PG_PASSWORD=
```

Comment:
lets add `export SKIP_EVENTS=1` as well

Reply:
see here as to why https://github.com/cmckee-dev/code_fund_ads/blob/1bfa9b1593c9da06241dda125af1f79c8a39bf54/app/models/concerns/eventable.rb#L27

Reply:
Will add this as well.